.pre-commit-config.yaml

```diff
@@ -5,17 +5,22 @@ repos:
     rev: v4.0.1
     hooks:
       - id: check-merge-conflict
+      - id: check-added-large-files
```

Comment:
I never want to add a file like the tailwind.css file again. Its easier to have this check which alerts the person than have to do force pushes to overwrite history.

Reply:
Yeah, but that will rarely happen and we can always just review the files.

Reply:
Better to prevent it from happening than to let it be commited. If it has to be added, a user can always disable a single hook for a commit, and this one in particular only checks new files.

Reply:
Right, again I am not sure if this is really needed, I don't have a problem with having it in the hooks nor I would really like to have it there.